src/main/java/com/iemr/common/controller/users/IEMRAdminController.java

```diff
@@ -208,6 +219,49 @@ public String userAuthenticate(
 		return response.toString();
 	}
 
+	@CrossOrigin()
```

Comment:
Cross origin is not needed.
If fact, there is a ticket files to fix this.
https://github.com/PSMRI/AMRIT/issues/47